x-pack/plugins/observability/public/hooks/use_fetcher.tsx

```diff
@@ -28,7 +28,7 @@ type InferResponseType<TReturn> = Exclude<TReturn, undefined> extends Promise<in
   : unknown;
 
 export function useFetcher<TReturn>(
```

Comment:
Should we delete the useFetcher hook in APM to avoid the two drifting apart again?

Reply:
Preferably yes! But it's kind of hard to share public code right now. Hoping we can tackle that later.